README.md

```diff
@@ -67,12 +100,23 @@ into IPLD graph in the following way:
     "email": "author@mail",
     "name": "Author Name"
   },
-  "type": "commit"
+  "tagType": "commit"
```

Comment:
Is this the only other change to the output data (i.e. breaking changes a user might need to know about)?

From what I can tell from the PR the reason for this change has little to do with UX and is due to some tooling problems on our part. We can potentially live with this, but it'd be helpful to know how painful the fix is.

Reply:
the tagType unfortunate side effect is mentioned here: https://github.com/ipfs/go-ipld-git/pull/46#issuecomment-884854758

perhaps @warpfork has thoughts. it seems to me like it's worth trying to undo this breaking change by improving our codegen before we merge this. I don't have the bandwidth for it today, but I could take a look over the weekend.

Reply:
Fairly sure this should be doable -> https://github.com/ipld/go-ipld-prime/issues/206#issuecomment-889903156

Reply:
I think the schema and code here is now fixed, but this documentation diff needs to be reverted now to match again :)